process/block/displayMetaBlock.go

```diff
@@ -88,19 +98,12 @@ func (hc *headersCounter) displayLogInfo(
 
 	log.Debug(message, arguments...)
 
-	numTxs := getNumTxs(header, body)
-	tps := numTxs / roundDuration
-	if tps > hc.peakTPS {
-		hc.peakTPS = tps
-	}
-
-	log.Debug("tps info",
+	log.Debug("metablock metrics info",
 		"shard", header.GetShardID(),
```

Comment:
I think shard is not needed to be displayed as well

Reply:
right. Will remove